spacy/tests/test_cli_app.py

```diff
@@ -40,3 +42,52 @@ def test_benchmark_accuracy_alias():
     assert result_benchmark.stdout == result_evaluate.stdout.replace(
         "spacy evaluate", "spacy benchmark accuracy"
     )
+
+
+def test_debug_data_trainable_lemmatizer_cli():
+    nlp = English()
```

Comment:
Minor, but you can use the fixture `en_vocab` instead for cases like this and it's slightly faster.

Reply:
Thanks for the suggestion, I wasn't aware we had a fixture for that. I've updated to use that instead.